src/app/services/sms/sms.factory.ts

```diff
@@ -31,9 +31,11 @@ export const createSmsFactory = (
     const errorMessage = 'SMS feature must be enabled in Feature Manager first'
     return {
       sendAdminContactOtp: () => {
+        //eslint-disable-next-line
         throw new Error(`sendAdminContactOtp: ${errorMessage}`)
```

Comment:
fyi, this will be changed to `Promise.reject` as part of #941

Reply:
great!